_posts/2022-05-04-Subnero's-highlights-at-SIWW2022-Newsletter.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: Subnero's highlights at SIWW 2022
+banner : images/pulse-SIWWSWAN.jpg
+date:  2019-10-15
+categories: newsletter
+thumbnail: images/pulse-thumbnail-marinaSWAN.jpg
```

Comment:
Can you make this image in square form factor please? so that it will be similar to the other square images? I will update the write up accordingly.

Reply:
Done! Updated the writeup to thumbnail: images/pulse-thumbnail-marinaSWAN2.jpg